src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -77,25 +77,24 @@ void OnSystemRequestNotification::Run() {
     return;
   }
 
-  if (RequestType::PROPRIETARY == request_type) {
-    /* According to requirements:
-       "If the requestType = PROPRIETARY, add to mobile API fileType = JSON
-        If the requestType = HTTP, add to mobile API fileType = BINARY"
-       Also in Genivi SDL we don't save the PT to file - we put it directly in
-       binary_data */
-
-    const std::string filename =
-        (*message_)[strings::msg_params][strings::file_name].asString();
-    BinaryMessage binary_data;
-    file_system::ReadBinaryFile(filename, binary_data);
+  const std::string filename =
+      (*message_)[strings::msg_params][strings::file_name].asString();
+  BinaryMessage binary_data;
+  file_system::ReadBinaryFile(filename, binary_data);
 #if defined(PROPRIETARY_MODE)
-    AddHeader(binary_data);
+  AddHeader(binary_data);
 #endif  // PROPRIETARY_MODE
 
 #if defined(PROPRIETARY_MODE) || defined(EXTERNAL_PROPRIETARY_MODE)
-    (*message_)[strings::params][strings::binary_data] = binary_data;
+  (*message_)[strings::params][strings::binary_data] = binary_data;
 #endif  // PROPRIETARY_MODE
 
+  if (RequestType::PROPRIETARY == request_type) {
+    /* According to requirements:
+       "If the requestType = PROPRIETARY, add to mobile API fileType = JSON
+        If the requestType = HTTP, add to mobile API fileType = BINARY"
+       Also in Genivi SDL we don't save the PT to file - we put it directly in
```

Comment:
@LitvinenkoIra please remove comments related to Genivi SDL. There is no knowledge in ope source about other sdl versions. And comment is not actual.

Reply:
@LuxoftAKutsan  Done c6ff051bab6089706dda993b11c8aa2e48856397